db/migrate/20210308092402_add_os_version_to_profiles.rb

```diff
@@ -0,0 +1,18 @@
+class AddOsVersionToProfiles < ActiveRecord::Migration[5.2]
+  def up
+    remove_index :profiles, name: 'uniqueness'
+    add_column :profiles, :os_version, :string, default: nil
```

Comment:
I've started thinking about how the API is going to look. We currently already have `os_major_version` which comes from the benchmark. If we add this field as `os_version` here, it's partially redundant. Perhaps we should only store `os_minor_version` instead. @vkrizan @skateman what do you think? There's a issue with data integrity if we allow setting any `os_version` as a string without validating that it also matches the `os_major_version` from the benchmark.

Reply:
Hmm, part of me is saying no to the suggestion and part is interested in the change. I would be more for a split into os_major and os_minor (two fields) to match models... If you think about it, our Policy models are not technically tied to an os_major. All of that is inferred from the one of the profile's benchmark and even that is inferred from the benchmarks ref_id. There are lots of parts that can go out of sync, so I would not worry about them much at this point. If we go that way, I'd suggest a refactoring to the policy model to have an os_major_version instead of the the inferred one. However, any of the solutions would work nominally. I'm inclined to consistency.

Reply:
I added the minor version field in a separate PR https://github.com/RedHatInsights/compliance-backend/pull/778

Reply:
I've realized that later... The minor on the profile and inferred major within benchmark make sense.

As the os_major_version is inferred from benchmark, combination of `os_minor_version` and `benchmark_id` is good enough.